apps/ui/src/views/Space/Editor.vue

```diff
@@ -214,11 +235,10 @@ async function handleProposeClick() {
       );
     } else {
       const appName = (route.query.app as LocationQueryValue) || '';
-      const currentTime = Math.floor(Date.now() / 1000);
-      const start = currentTime + props.space.voting_delay;
-      const minEnd = start + props.space.min_voting_period;
-      const maxEnd = start + props.space.max_voting_period;
 
+      // Proposal start, min and end time are unix timestamp,
+      // are are not compatible with offchain spaces
+      // (these args are ignored by evm networks)
```

Comment:
```suggestion
      // and are not compatible with onchain EMV spaces (those use blocks instead of timestamps)
      // (these args are ignored by onchain networks)
```